GuideForNewDevelopers.md

```diff
@@ -0,0 +1,579 @@
+      - [2.1.10 Start the server(Finally)](#2110-start-the-serverfinally)
+    - [2.2 Windows](#22-windows)
+      - [2.2.1 Install Python3](#221-install-python3)
+      - [2.2.2 Install git Bash](#222-install-git-bash)
+      - [2.2.3 Install VSCode](#223-install-vscode)
+      - [2.2.4 Clone the Project Locally](#224-clone-the-project-locally)
+      - [2.2.5 Install and Create Virtual Environment](#225-install-and-create-virtual-environment)
+        - [2.2.5.1 Create a Virtual Environment](#2251-create-a-virtual-environment)
+        - [2.2.5.2 Activate the Virtual Environment](#2252-activate-the-virtual-environment)
+      - [2.2.6 Install requirements](#226-install-requirements)
+      - [2.2.7 Run the project](#227-run-the-project)
+        - [2.2.7.1 Open CMD and navigate inside the project directory](#2271-open-cmd-and-navigate-inside-the-project-directory)
+        - [2.2.7.2  Activate virtual python environment.](#2272--activate-virtual-python-environment)
+        - [2.2.7.3 Copy .env.example file to .env](#2273-copy-envexample-file-to-env)
+        - [2.2.7.4 Run the following commands to setup the database](#2274-run-the-following-commands-to-setup-the-database)
+      - [2.2.8 Start the server(Finally)](#228-start-the-serverfinally)
+    - [2.3 Linux](#23-linux)
+      - [2.3.1 Install Python3](#231-install-python3)
+      - [2.3.2 Install git](#232-install-git)
+      - [2.3.3 Install sqlite3](#233-install-sqlite3)
+      - [2.3.4 Install VSCode](#234-install-vscode)
+      - [2.3.5 Clone the Project Locally](#235-clone-the-project-locally)
+      - [2.3.6 Install and Create Virtual Environment](#236-install-and-create-virtual-environment)
+        - [2.3.6.1 Create a Virtual Environment](#2361-create-a-virtual-environment)
+        - [2.3.6.2 Activate the Virtual Environment](#2362-activate-the-virtual-environment)
+      - [2.3.7 Install requirements](#237-install-requirements)
+      - [2.3.8 Run the project](#238-run-the-project)
+        - [2.3.8.1 Open a terminal and navigate inside the project directory](#2381-open-a-terminal-and-navigate-inside-the-project-directory)
+        - [2.3.8.2 Activate virtual python environment.](#2382-activate-virtual-python-environment)
+        - [2.3.8.3 Copy .env.example file to .env](#2383-copy-envexample-file-to-env)
+        - [2.3.8.4 Run the following commands to setup the database](#2384-run-the-following-commands-to-setup-the-database)
+      - [2.3.9 Start the server(Finally)](#239-start-the-serverfinally)
+  - [Working on new features](#working-on-new-features)
+  - [Troubleshooting[WIP]](#troubleshootingwip)
+    - [MAC](#mac)
+      - [Homebrew not found](#homebrew-not-found)
+    - [Windows](#windows)
+    - [Linux](#linux)
+
+
+# Guide for New Developers
+
+## 1. Summary of Development Setup
+
+The Development Environment setup requires the use of the following tools/software
+
+1. Package Management System
```

Comment:
Here the other PMS like YUM, Aptiude and homebrew are PMS for Operating system, Instead of pip or Anaconda, 

Do you think something like [Chocolatey](https://chocolatey.org/) might be better ?@joamats

Reply:
But not sure if is worth mentioning chocolatey as the guide doesn't use it. 

On other note, because windows is giving issues with uWSGI, i am also updating the guide to use WSL2 in windows